doc/nrf/known_issues.rst

```diff
@@ -19,7 +19,8 @@ Use the drop-down filter to see known issues for previous releases and check if
 
```

Comment:
This change is also needed in https://github.com/nrfconnect/sdk-nrf/pull/3376
In 1.4.1, this page is orphaned, but it is still included (and customers might end up on it by switching versions). It would be confusing to not have the 1.4.1 issues tagged there.

Reply:
Thanks, this is done in #3376 .